app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -307,6 +312,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
     }
 
     final override fun onPause() {
+        // We should return to the browser if there were normal tabs when we left the app
+        settings().shouldReturnToBrowser =
+            components.core.store.state.getNormalOrPrivateTabs(private = false).isNotEmpty()
```

Comment:
There's one case we're not handling here: When the user exits the app from home, we would still return them to the a tab (if one exists), but we should probably return to home in this case, e.g. User has a few tabs, opens home via "+" in the tabs tray, exits app. What do you think? Should we cover that?

Reply:
I think this could be up to UX but I personally think it makes sense to treat "home" as the ephemeral new tab state that I wouldn't want to be returned to. Like if you went to settings, you wouldn't expect to be returned to settings on a cold start.

I think a larger discussion could happen around this interaction, and I also have brought up to UX that we could make this a Setting - ie launch to home/launch to browser with tabs.

Reply:
> I personally think it makes sense to treat "home" as the ephemeral new tab state that I wouldn't want to be returned to.

Interesting, yes. I think we can land and refine later. The reason I brought it up is because it's not necessarily clear which tab to restore, for example:

- open a few normal tabs and select one
- open a private tab
- restart the app
- now user is in private mode with no open tab (Good)
- exit private mode
- now user is on home screen (Good)
- restart the app
- now user is on newest normal tab as no tab was selected anymore (this seems fine, but launching back to home seemed better to me?)